AlphaWallet/Tokens/Collectibles/ViewModels/NFTAssetViewModel.swift

```diff
@@ -105,10 +106,14 @@ class NFTAssetViewModel {
     func transform(input: NFTAssetViewModelInput) -> NFTAssetViewModelOutput {
         let whenOpenSeaStatsHasChanged = PassthroughSubject<Void, Never>()
         if let collectionId = tokenHolder.values.collectionId, collectionId.trimmed.nonEmpty {
-            nftProvider.collectionStats(collectionId: collectionId).done { [weak self] stats in
-                self?.configure(overiddenOpenSeaStats: stats)
-                whenOpenSeaStatsHasChanged.send(())
-            }.cauterize()
+            nftProvider.collectionStats(collectionId: collectionId)
+                .receive(on: RunLoop.main)
```

Comment:
Shouldn't upstream be already on `.main` by convention? If it isn't, can we make it so in a separate PR?